### PR TITLE
Single Page App Milestone :green_circle: 

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
- In this project, my partner @cyrilmuchemi and I have added pages to our book apps.
- The pages load without refreshing the browser (Single Page App) and fetches the pages from the server instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ahmetbozaci) [inline README.md]: - [ ] You wrapped all sections as a code. Please update this part and your readme file look professional. 

image

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+# framework-awesomebook
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesomebook
+
+> This a group work project
+
+# WORKFLOW SCREENSHOT
+![screenshot]()
+
```
Comment (ahmetbozaci) [inline README.md:10]: Please fix the broken link in the screenshot.

File: README.md [hunk 3]
```diff
@@ -1 +1,65 @@
-# framework-awesomebook
\ No newline at end of file
+# framework-awesomebook
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesomebook
+
+> This a group work project
+
```
Comment (ahmetbozaci) [inline README.md]: - [ ] Please add a description of the project.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+header div:nth-child(2) nav {
+  margin: 10px 15px;
+}
+
+header div:nth-child(2) ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
```
Comment (ahmetbozaci) [inline style.css:62]: ### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @SiansandaTamara @afikanzoro  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your Github flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
@SiansandaTamara 
- You have one commit for HTML, CSS, and js files. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: README.md [hunk 5]
```diff
@@ -1 +1,61 @@
...
+- Web Browser
+- Code Editor
+- Prior knowledge of CSS, HTML, GIT, GITHUB and Linters.
+
+### Setup
+
+```bash
+git clone// https://siansandatamara.github.io/awesome-book/
+cd awsomebook
+
```
Comment (CollinsTatang) [inline README.md:34]: ### **OPTIONAL**
- [ ] Kindly replace the below code to solve the problem with your readme file. You opened a special character to write a block of code and you didn't close it for that reason any other text below it is affected.
bash

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**